src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/nuget.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Since we are using dotnet restore at the tool-runtime directory to restore the runtime, dnu will grab the nuget.config file in the working directory to get the feeds. If we don't want to add this file, I could also pass in the --source params to the dotnet restore command.